stable/traefik/templates/configmap.yaml

```diff
@@ -13,7 +13,7 @@ data:
     {{- if .Values.debug.enabled }}
     debug = true
     {{- else }}
-    logLevel = "INFO"
+    logLevel = "{{ .Values.logLevel }}"
```

Comment:
I don't have a problem with this, but can you default this to `INFO` for backwards compatibility?

Reply:
https://helm.sh/docs/chart_template_guide/#using-the-default-function

default INFO :)